pkg/apis/serving/v1alpha1/service_validation.go

```diff
@@ -180,3 +181,24 @@ func (rt *ReleaseType) Validate(ctx context.Context) *apis.FieldError {
 
 	return errs.Also(rt.Configuration.Validate(ctx).ViaField("configuration"))
 }
+
+func (s *Service) validateCreatorAndModifier(ctx context.Context) *apis.FieldError {
+	if len(s.GetAnnotations()) == 0 {
+		return nil
+	}
+	var errs *apis.FieldError
+	if apis.IsInUpdate(ctx) {
+		original := apis.GetBaseline(ctx).(*Service)
+		if original.GetAnnotations()[serving.CreatorAnnotation] != s.GetAnnotations()[serving.CreatorAnnotation] {
+			errs = errs.Also(&apis.FieldError{
+				Message: fmt.Sprintf("update to " + serving.CreatorAnnotation + " annotation is immutable"),
```

Comment:
you're not formatting anything, so just remove the `fmt.Sprintf`
Also just `annotatio + " is immutable"`.

Reply:
Same below.